telemetry/definitions/commonDefinitions.json

```diff
@@ -537,6 +543,57 @@
             "description": "The Toolkit has completed initialization",
             "metadata": [{ "type": "duration" }],
             "passive": true
+        },
+        {
+            "name": "sqs_openQueue",
+            "description": "Open an SQS queue. Initially opens to either the send message pane or poll messags pane.",
+            "metadata": [{ "type": "sqsQueueType" }]
+        },
+        {
+            "name": "sqs_createQueue",
+            "description": "Create a new SQS queue",
+            "metadata": [{ "type": "result" }, { "type": "sqsQueueType" }]
+        },
+        {
+            "name": "sqs_deleteQueue",
+            "description": "Delete a SQS queue",
+            "metadata": [{ "type": "result" }, { "type": "sqsQueueType" }]
+        },
+        {
+            "name": "sqs_sendMessage",
+            "description": "Send a message to an SQS queue",
+            "metadata": [{ "type": "result" }, { "type": "sqsQueueType" }]
+        },
+        {
+            "name": "sqs_deleteMessages",
+            "description": "Delete one or more messages from an SQS queue. Value indicates the number of messages that we tried to delete.",
+            "unit": "Count",
+            "metadata": [{ "type": "result" }, { "type": "sqsQueueType" }]
+        },
+        {
+            "name": "sqs_copyMessage",
+            "description": "Copy a message to the clipboard",
+            "metadata": [{ "type": "sqsQueueType" }]
```

Comment:
we might instead want a generic `copyToClipboard` metric with perhaps a parameter that indicates the component that initiated it. 

Though having telemetry on copy/paste in general seems a bit much, almost like tracking how many keys the user pressed...

Reply:
Yeah, maybe we should just not have this. It doesn't really tell us anything, removed.